imports/plugins/core/shipping/client/containers/parcelSizeSettings.js

```diff
@@ -0,0 +1,115 @@
+import { Meteor } from "meteor/meteor";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Reaction, i18next } from "/client/api/";
+import { Shops } from "/lib/collections";
+import { ShippingParcel } from "/lib/collections/schemas";
+import { convertWeight, convertLength } from "/lib/api";
+import ParcelSizeSettings from "../components/parcelSizeSettings";
+
+/**
+ * @method saveDefaultSize
+ * @summary call "shipping/updateParcelSize" method
+ * @param {Object} size - size object to be saved
+ * @since 2.0.0
+ * @return {Function} callback
+*/
+const saveDefaultSize = (size) => {
+  const parcel = ShippingParcel.clean(size);
+  Object.keys(parcel).forEach((key) => {
+    parcel[key] = Number(parcel[key]);
+  });
+  try {
+    ShippingParcel.validate(parcel);
```

Comment:
We are just catching this and throwing it away?

Reply:
Fixed